src/frontend/src/app/asset-readings/assets/assets.component.html

```diff
@@ -81,7 +81,7 @@ <h5 class="title"><small> Asset Readings </small></h5>
                     {{d.reading | json}}
                   </td>
                   <td>
-                    <span class="tag is-rounded">{{d.timestamp | date: 'dd-MM-yyyy HH:mm:ss'}}</span>
+                    <span class="tag is-rounded">{{d.timestamp | moment: 'DD/MM/YYYY HH:mm:ss'}}</span> 
```

Comment:
`-` to `/` ? in TH can we add a tooltip (or light color text with format help)

Reply:
Yes, we can add a tooltip. I will add it in the next commit.